src/runtime/client-event-manager.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+/* eslint-disable */
```

Comment:
Why is this necessary?

Reply:
I'll remove it. It was added by mistake. I deleted before pushing, but the file was not automatically saved. Good catch!